src/com/maddyhome/idea/vim/helper/EditorHelper.java

```diff
@@ -521,7 +521,7 @@ public static CharBuffer getLineBuffer(@NotNull final Editor editor, final int l
   public static boolean isLineEmpty(@NotNull final Editor editor, final int line, final boolean allowBlanks) {
     CharSequence chars = editor.getDocument().getCharsSequence();
     int offset = getLineStartOffset(editor, line);
-    if (chars.charAt(offset) == '\n') {
+    if (chars.length() == 0 || chars.charAt(offset) == '\n') {
```

Comment:
Could you please substitute this check with the check for `offset` being within the limits of `chars`'s size? It would be a more reliable fix.